examples/check-semantics/check_semantics.description

```diff
@@ -1,8 +1,8 @@
-// `check` semantics is not simply to return on error.
-// When `check` gets an error, it fails.
-// Enclosing block decide how to handle failure.
-// Most blocks pass failure up to enclosing block.
-// Function definition handles failure by returning the error.
-// `on fail` can catch the error.
-// `fail` statement is like `check` but always fails.
-// Differs from exceptions in that control flow is explicit
+// `check` semantics is simply not to return on an `error`.
```

Comment:
Wasn't not in the correct place?

Same comment re: emphasizing error.

Reply:
didn't get what you mean by  `Wasn't in the correct place?`. Please check with the new changes.

Reply:
Sorry, meant to say the previous version was correct.

Maybe
```
// `check` semantics is not simply to return on an `error` value.
```

Reply:
reading `not simply to return` felt a bit odd to me. Anyway, updated according to the above suggestion. thanks.

Reply:
Or actually maybe
```
// `check` semantics is not to simply return on an `error` value.
```

@praneesha, wdyt? What we want to say is "`check` doesn't just mean returning when the expression evaluates to error".

Reply:
+1 on the new suggestion. Let's see if praneesha has anything to add.

Reply:
Resolving the conversation as the new suggestion was added.